src/components/create-safe/steps/SetNameStep.tsx

```diff
@@ -52,7 +52,7 @@ const SetNameStep = ({ params, onSubmit, onBack, setStep }: Props) => {
           <Box padding={3}>
             <Typography variant="body1" mb={2}>
               You are about to create a new Safe wallet with one or more owners. First, let&apos;s give your new wallet
-              a name. This name is only stored locally and will never be shared with Gnosis or any third parties. The
+              a name. This name is only stored locally and will never be sent to the backend or any third parties. The
```

Comment:
I think "shared with Safe" or "shared with us" is a nicer phrasing. It's less technical.

Reply:
Same goes for `<SetAddressStep>`.

Reply:
And `<OwnerList>`.